packages/front-end/components/Tooltip.tsx

```diff
@@ -4,32 +4,50 @@ import { MdInfoOutline } from "react-icons/md";
 // todo: add support for variable positioning instead of just bottom.
 
 interface Props extends HTMLAttributes<HTMLDivElement> {
-  text: string;
+  body: string | JSX.Element;
   tipMinWidth?: string;
+  tipTriggerPosition?: "position-fixed";
   tipPosition?: "bottom" | "top" | "left" | "right";
   innerClassName?: string;
+  textColor?: string;
+  bgColor?: string;
+  theme?: "white";
```

Comment:
If we're doing theme, do we need text and bg color too?  Personally, I would just do a `theme` prop since that will ensure the app stays visually consistent.  We don't want 10 shades of blue being used.

```suggestion
  theme: "blue" | "white";
```

So setting the theme would also set appropriate bg and text colors.